enclave/core/sgx/entropy.c

```diff
@@ -50,10 +40,7 @@ int mbedtls_hardware_poll(
             *p++ = *q++;
```

Comment:
What is q here? Is it a global variable? I am trying to understand this and dont see q defined in this function.

Reply:
The GitHub diff is bad and cut it out, it's a few lines above:

> const unsigned char* q = (const unsigned char*)&x;

Reply:
@andschwa  Thanks :)